coalib/misc/Constants.py

```diff
@@ -13,6 +13,11 @@
                  "happen. When asked for, the following information "
                  "may help investigating:")
 
+VERSION_CONFLICT_MESSAGE = ("There is a conflict in the version of a package "
+                            "you have installed and the requirements of "
+                            "coala. The following information may help "
+                            "to resolve this conflict: ")
```

Comment:
If you want to give a better help , maybe https://pypi.python.org/pypi/pip-conflict-checker/0.3 should be mentioned ?